products.html

```diff
@@ -0,0 +1,640 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel = "stylesheet" href = "https://liniui.netlify.app/component.css" /> 
+    <link rel="stylesheet" href="./products.css">
+    <title>Document</title>
+</head>
+<body>
+    <div class="container product-navbar">
+        <div class="navbar">
+            <div class="logo">
+             vivir
```

Comment:
You can put the home page link in the logo.

Reply:
made the changes you suggested, please approve the PR